recipes/django-guardian/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-guardian" %}
+{% set version = "1.4.8" %}
+{% set sha256 = "353794b2f685c588e44583e18dcc2bf5fc33b0870a344ab7a1bb00764877350d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
You need to add `pytest-runner` as a `build` requirement since it's listed in the `setup_requires`:

https://github.com/django-guardian/django-guardian/blob/devel/setup.py#L15